### PR TITLE
Rework Hugo variables, implement default canonical links

### PR DESCRIPTION
**Pull request by TheOnlyRew**

### CODE REVIEW

**Comment by laurenrother** [comment]
👍

File: config.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ params:
   #
   # 1. The project name must match some contents' `project:` front matter
   # 2. The `path` is the on-disk relative path from the content root. In Hugo,
-  #    the content root is 'content/', on S3 the root is simply '/'.
+  #    the content root is '/content/', on S3 the root is simply '/'.
   #    If the project was moved from the Middleman site to Hugo, the
   #    `arichved_path` is the on-disk relative path of the previous directory.
```
Comment (erjohnson) [inline config.yaml]: archived, nbd though.

File: layouts/partials/head.html [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
-  <meta name="version_url" content='/{{ .Scratch.Get "version_url" }}'>                  <!-- ex; /riak/kv/2.1.4 -->
-  <meta name="project_relative_url" content='{{ .Scratch.Get "project_relative_url" }}'> <!-- ex; installing/ -->
-
-{{ if .Params.version_history.in }}
-  <meta name="version_history_in" content="{{ .Params.version_history.in }}">
-{{ end }}
-{{ if .Params.version_history.locations }}
-  <meta name="version_history_locations" content="{{ .Params.version_history.locations | jsonify }}">
-{{ end }}
+  <title>{{ .Title }}</title>
```
Comment (erjohnson) [inline layouts/partials/head.html]: maybe tack in:

```css
<title>{{ if .Params.title_supertext }}{{ .Params.title_supertext }}: {{ end }}{{ .Title }}</title>
```

**Comment by erjohnson** [comment]
aside from my comments 💯 👍

File: layouts/partials/head.html (hunk 2 above)
Comment (TheOnlyRew) [inline layouts/partials/head.html]: Good catch. Wound up going with the below (w/o the ":") b/c some (all?) of our supertext elements explicitly include a trailing ":", and I don't want duplicates.

```
  <title>{{ if .Params.title_supertext }}{{ .Params.title_supertext }} {{ end }}{{ .Title }}</title>
```

**Pull request merged**